Classes/commands.py

```diff
@@ -1106,3 +1106,161 @@ async def count_officers(self, ctx):
 
         # Send the results
         await ctx.channel.send(embed=embed)
+
+    @checks.is_event_bot_channel()
+    @checks.is_event_host_or_any_trainer()
+    @commands.command()
```

Comment:
One extra thing, when using *args, you can add usage="how to use command" in order to not show [*args] on the help menu and instead show your own constructed usage, like usage="[options] <officer>" for the =patrol_time command.

Reply:
i need to look into it more because the only case i could find where it should display the full help is when you do `=who -h`
the problem is that it doesn't work as it tries to exit the program and we catch exception `SystemExit`

Reply:
never mind i miss understood

Reply:
fixed f3c63fe11b605fac0900ce5d1b64c483b5deae39